packages/toolpad-app/src/toolpad/AppEditor/BindingEditor.tsx

```diff
@@ -241,8 +241,9 @@ export function ValueBindingEditor({ value, onChange, error }: ValueBindingEdito
             }}
           >
             <Typography sx={{ mb: 2, color: 'red' }}>
-              Error while reading the prettier configuration: The prettier config could not be
-              loaded and therefore the code would not be formatted
+              Error while reading the prettier configuration:
+              {(error as string) ??
```

Comment:
The `error` prop is `unknown`, casting it to a `string` potentially results in undesired behavior. `@mui/toolpad-utils/errors` has this handy `errorFrom` function which parses any input into an error object. We use it sometimes to [extract error messages](https://github.com/mui/mui-toolpad/blob/9596a8fe03c5145597ecd337ac336dcfb0ddd0f9/packages/toolpad-components/src/components/ErrorOverlay.tsx#L24) from `unknown` input.